AccountService.js

```diff
@@ -27,17 +27,76 @@ export class AccountService {
     return false;
   }
 
-  async create({url = this.config.urls.base, email}) {
-    const response = await axios.post(url, {email}, {
-      headers: {'Accept': 'application/ld+json, application/json'}
-    });
+  async create({baseUrl = this.config.urls.base, email}) {
```

Comment:
Using `baseUrl` here is a bit weird -- it makes sense in the other cases because it's understood that the other parameters will be mixed with it. But that's not true here. I'm hesitant to request this be changed back to `url` since all of the other calls use `baseUrl`... but that's really the only reason. We can leave it as is and fix it up later if it's a real point of confusion.

Reply:
no issue fixing it